<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QRenderingOptions</class>
 <widget class="QDockWidget" name="QRenderingOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>313</width>
    <height>210</height>
   </rect>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="windowTitle">
   <string>Rendering options</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Layers</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="m_qSSAOCheck">
         <property name="text">
          <string>SSAO</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_qDiffuseCheck">
         <property name="text">
          <string>Diffuse</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_qSpecularCheck">
         <property name="text">
          <string>Specular</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_qColorCheck">
         <property name="text">
          <string>Color</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_qFXAACheck">
         <property name="text">
          <string>FXAA</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_qGridCheck">
         <property name="text">
          <string>Grid</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Colors</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="m_qNylon">
         <property name="text">
          <string>Nylon</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_qExposure">
         <property name="text">
          <string>Exposure</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QCheckBox" name="m_qFrame">
           <property name="text">
            <string>Frame</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>Color...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QCheckBox" name="m_qLut">
           <property name="text">
            <string>Lut</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>Load...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionUpdateLayers">
   <property name="text">
    <string>UpdateLayers</string>
   </property>
  </action>
  <action name="actionUpdate_Post">
   <property name="text">
    <string>Update Post</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_qSSAOCheck</sender>
   <signal>stateChanged(int)</signal>
   <receiver>actionUpdateLayers</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_qDiffuseCheck</sender>
   <signal>stateChanged(int)</signal>
   <receiver>actionUpdateLayers</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_qSpecularCheck</sender>
   <signal>stateChanged(int)</signal>
   <receiver>actionUpdateLayers</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_qColorCheck</sender>
   <signal>stateChanged(int)</signal>
   <receiver>actionUpdateLayers</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_qFXAACheck</sender>
   <signal>stateChanged(int)</signal>
   <receiver>actionUpdateLayers</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_qGridCheck</sender>
   <signal>stateChanged(int)</signal>
   <receiver>actionUpdateLayers</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_qNylon</sender>
   <signal>stateChanged(int)</signal>
   <receiver>actionUpdate_Post</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>208</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_qExposure</sender>
   <signal>stateChanged(int)</signal>
   <receiver>actionUpdate_Post</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>208</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_qFrame</sender>
   <signal>stateChanged(int)</signal>
   <receiver>actionUpdate_Post</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>168</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_qLut</sender>
   <signal>stateChanged(int)</signal>
   <receiver>actionUpdate_Post</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>168</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
